common/cpu/intel/default.nix

```diff
@@ -12,4 +12,11 @@
     libvdpau-va-gl
     intel-media-driver
   ];
+
+  hardware.opengl.extraPackages32 = with pkgs.pkgsi686Linux; [
```

Comment:
Since 32bit is phasing out, should this be only added if `hardware.opengl. driSupport32Bit` is set?

Reply:
That would be strange I think. `extraPackages32` is only referred to if `driSupport32Bit` is set. Functionally there will be no change if `extraPackages32` is set while `driSupport32Bit` is not.

See https://github.com/NixOS/nixpkgs/blob/0699530f08290f34c532beedd66046825d9756fa/nixos/modules/hardware/opengl.nix#L142-L143 and https://github.com/NixOS/nixpkgs/blob/0699530f08290f34c532beedd66046825d9756fa/nixos/modules/hardware/opengl.nix#L18-L21

Reply:
Ok. If that is the case, than we don't need to make it conditional.